gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -327,21 +327,25 @@ on_entry_icon_press (GtkEntry            *entry,
   if (icon_pos != GTK_ENTRY_ICON_SECONDARY)
     return;
 
+  /* We show the eye icon with a slash through it when the password is not
+   * visible, which is consistent with how gnome-shell works in
+   * StPasswordEntry.
+   */
   if (gtk_entry_get_visibility (entry))
```

Comment:
We need to reverse the check here also no?

Reply:
No, then we'd be back to where we started. A double negative is a positive

Reply:
Never mind, you're right actually